### PR TITLE
Add Music Fix, resolves issue #177

### LINKED ISSUE #177: APlayer stuck in collapsed after adding music
## Steps to Reproduce: 

1. Click `addMusic(...)`
1. Click APlayer menu icon `aplayer-icon-menu`
1. Click `addMusic(...)`
1. Click APlayer menu icon `aplayer-icon-menu`

**Result:** APlayer is stuck in collapsed
**What should happen:** APlayer toggles between `aplayer-list` and `aplayer-list aplayer-list-hide`

## Example 
aplayeraddmusicbug

### PR DESCRIPTION
**Pull request by MPierre9**
Fixes #177 the `addMusic(newMusic)` function by allowing APlayer to toggle between `aplayer-list` and `aplayer-list aplayer-list-hide` with animation when adding music.

## Result
aplayeraddmusicfix

### CODE REVIEW

File: src/APlayer.js [hunk 1]
```diff
@@ -940,10 +940,16 @@ class APlayer {
             this.element.classList.add('aplayer-withlist');
             this.audio.loop = false;
         }
+        var songListLength = listEle.getElementsByTagName('li').length;
 
-        list.style.height = 'auto';
-        list.style.height = list.offsetHeight + 'px';
-
+        if(list.classList.contains('aplayer-list-hide')) {
+            list.style.height = (songListLength * 32.75) + 'px';
```
Comment (micooz) [inline src/APlayer.js]: Does `32.75` make sense?

Comment (MPierre9) [reply src/APlayer.js]: You're right, rounded up to 33px.

**Comment by DIYgod** [comment]
I find there is no need to distinguish whether list is hidden, `list.style.height = (songListLength * 33) + 'px'` can meet all conditions.

**Comment by MPierre9** [comment]
@DIYgod Thanks for pointing that out, I thought the bug only occurred when the list was hidden. Implementing your change allows the addMusic animation to run when APlayer is in list which is great 👍

**Pull request merged**